sdk/eventhub/azure-eventhub/stress/logger.py

```diff
@@ -76,3 +79,12 @@ def get_logger(log_filename, logger_name, level=logging.INFO, print_console=Fals
         stress_logger.addHandler(stress_file_handler)
 
     return stress_logger
+
+
+def get_azure_logger(logger_name, level=logging.INFO):
+    logger = logging.getLogger("azure_logger_" + logger_name)
+    logger.setLevel(level)
+    logger.addHandler(AzureLogHandler(
+        connection_string=os.environ['APP_INSIGHTS_CONN_STR'])
```

Comment:
let's make the env variable to APPLICATIONINSIGHTS_CONNECTION_STRING

- that way, we will not need to pass a connection string - OC exporter looks for the above env variable automatically ^^

Reply:
thanks for the breadcrumb, I double check the code, yes, it's written this way.